blocks/header-nav-chain-block/chains/header-nav-chain-block/_children/horizontal-links/default.test.jsx

```diff
@@ -67,7 +62,8 @@ describe('the links bar feature for the default output type', () => {
     );
 
     expect(wrapper.html()).toMatchInlineSnapshot(
-      '"<nav class=\\"horizontal-links-bar\\"><span class=\\"sc-bwzfXH dtelAW horizontal-links-menu\\">    <a href=\\"id_1/\\" class=\\"sc-bdVaJa jLoscj\\">test link 1</a></span></nav>"',
+      '"<nav class=\\"horizontal-links-bar\\" aria-label=\\"[object Object]\\"><span class=\\"sc-bwzfXH dtelAW horizontal-links-menu\\">    <a href=\\"id_1/\\" class=\\"sc-bdVaJa jLoscj\\">test link 1</a></span></nav>"',
```

Comment:
One thing to note with inline snapshots, when you update them via the jest cli it ends up with duplicate.. You can delete the first item here, and in the ones below